2-mandatory/4-restaurant.js

```diff
@@ -20,8 +20,31 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
-}
+  orderBurger(balance) {
```

Comment:
This looks good and works but this is the shorthand approach of adding a method into an object, just to check understanding, do you know how you would define this method without using the shorthand approach? Ie by defining fully the key and the value separately?